src/main/java/org/amahi/anywhere/activity/AuthenticationActivity.java

```diff
@@ -23,7 +23,9 @@
 import android.accounts.AccountAuthenticatorActivity;
 import android.accounts.AccountManager;
 import android.app.Activity;
+import android.content.Context;
 import android.graphics.PorterDuff;
+import android.media.Image;
 import android.os.Bundle;
 import android.support.design.widget.TextInputLayout;
```

Comment:
Please always clean the unused dependencies. Here two of them are unused!
`import android.media.Image;`
`import android.support.design.widget.TextInputLayout;`